go-grpc-gateway/test.sh

```diff
@@ -26,7 +28,7 @@ trap my_exit EXIT INT HUP TERM
 set -x
 
 # Spawn rust stub server
-./../target/debug/stub --client-listen-uri insecure-fog://localhost:3000 &
+"$CARGO_TARGET_DIR/debug/stub" --network-id "local" --client-listen-uri insecure-fog://localhost:3000 &
```

Comment:
network-id?

Reply:
this should be chain-id, thanks

Reply:
Maybe i can get circle CI to pass if I i keep it as `./../target/debug/stub` ?